src/radical/pilot/agent/scheduler/continuous.py

```diff
@@ -326,14 +350,16 @@ def _alloc_mpi(self, cud):
         'threads_per_proc', as otherwise the application would not be able to
         spawn the requested number of threads on the respective node.
         '''
-
+        
```

Comment:
For some reason, most people (and checkers) dislike spaces at the end of lines, like on the empty line above.  Don't ask me why, but there you go...